types/selenium-webdriver/devtools/networkinterceptor.d.ts

```diff
@@ -17,7 +17,7 @@ declare class HttpResponse {
      * @param {*} value
      */
     addHeaders(header: string, value: any): void;
-    get headers(): Array<{name: string, value: any}>;
+    get headers(): Array<{ name: string; value: any }>;
```

Comment:
Are semi-colons allowed between args?

Reply:
Between args, no. But this is inside an object literal (`{ ... }`).

Reply:
🤦 I misread the code. Thanks for explaining